src/faebryk/libs/app/parameters.py

```diff
@@ -16,10 +16,15 @@
 
 logger = logging.getLogger(__name__)
 
+MIF_TRAIT = F.is_dynamic_by_connections_alias | F.is_dynamic_by_connections_sum
+
 
 def resolve_dynamic_parameters(graph: Graph):
     other_dynamic_params, connection_dynamic_params = partition(
-        lambda param_trait: isinstance(param_trait[1], F.is_dynamic_by_connections),
```

Comment:
What about a hierarchy like

```python
class is_dynamic(Trait): ...

class is_dynamic_by_connection(is_dynamic): ...
# cut here and use `decless` if we made the generic version

class is_dynamic_by_connections_alias(is_dynamic_by_connection.impl()): ...

class is_dynamic_by_connections_sum(is_dynamic_by_connection.impl()): ...
```

The `isinstance` here is a bit of a smell, and very traitorous